templates/linux-template.j2

```diff
@@ -14,11 +15,12 @@ objects:
   metadata:
     name: ${NAME}
   spec:
+    running: false
     template:
       spec:
         domain:
           cpu:
-            cores: ${CPU_CORES}
+            cores: ${{'{{'}}CPU_CORES{{'}}'}}
```

Comment:
For me it was enough to provide quotes [here](https://github.com/ansibleplaybookbundle/import-vm-apb/blob/master/templates/linux-template.j2#L45)

Reply:
Since this is a j2 template you need to escape the {{ and }}.  According to the ocp template documentation you need to use ${{}} when you want an unquoted value.